test/interactions/panZoomInteractionTests.ts

```diff
@@ -111,238 +184,316 @@ describe("Interactions", () => {
+        xScale = new Plottable.Scales.Linear();
+        xScale.domain([0, SVG_WIDTH / 2]);
+
+        svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
+
+        let component = new Plottable.Component();
+        component.renderTo(svg);
+
+        panZoomInteraction = new Plottable.Interactions.PanZoom();
+        panZoomInteraction.addXScale(xScale);
+        panZoomInteraction.attachTo(component);
+
+        eventTarget = component.background();
+      });
+
+      it("can set minDomainExtent", () => {
+        let minimumDomainExtent = SVG_WIDTH / 4;
+        assert.strictEqual(panZoomInteraction.minDomainExtent(xScale, minimumDomainExtent), panZoomInteraction,
+          "setting the minDomainExtent returns the interaction");
+
+        assert.strictEqual(panZoomInteraction.minDomainExtent(xScale), minimumDomainExtent,
+          "returns the correct minDomainExtent");
+
+        svg.remove();
       });
 
-      it("Rejects negative extents", () => {
-        assert.throws(() => panZoomInteraction.minDomainExtent(xScale, -1), Error);
+      it("rejects negative extents", () => {
+        (<any>assert).throws(() => panZoomInteraction.minDomainExtent(xScale, -1), Error, "extent must be non-negative",
+          "Correctly rejects -1");
         svg.remove();
       });
 
-      it("can't be larger than maxDomainExtent() for the same Scale", () => {
-        let maximumDomainExtent = minimumDomainExtent * 2;
+      it("can't set minDomainExtent() be larger than maxDomainExtent() for the same Scale", () => {
+        let maximumDomainExtent = SVG_WIDTH / 2;
         panZoomInteraction.maxDomainExtent(xScale, maximumDomainExtent);
+
         let tooBigMinimumDomainExtent = maximumDomainExtent * 2;
-        assert.throws(() => panZoomInteraction.minDomainExtent(xScale, tooBigMinimumDomainExtent), Error);
+        (<any>assert).throws(() => panZoomInteraction.minDomainExtent(xScale, tooBigMinimumDomainExtent), Error,
+          "minDomainExtent must be smaller than maxDomainExtent for the same Scale",
+          "cannot have minDomainExtent larger than maxDomainExtent");
         svg.remove();
       });
 
-      it("Mousewheeling in cannot go beyond the specified domainExtent", () => {
+      it("cannot go beyond the specified domainExtent (mousewheel)", () => {
         // HACKHACK PhantomJS doesn't implement fake creation of WheelEvents
         // https://github.com/ariya/phantomjs/issues/11289
-        if ( window.PHANTOMJS ) {
+        if (window.PHANTOMJS) {
           svg.remove();
           return;
         }
 
+        let minimumDomainExtent = SVG_WIDTH / 4;
+        panZoomInteraction.minDomainExtent(xScale, minimumDomainExtent);
+
         let scrollPoint = { x: SVG_WIDTH / 4, y: SVG_HEIGHT / 4 };
         let deltaY = -3000;
 
-        TestMethods.triggerFakeWheelEvent("wheel", svg, scrollPoint.x, scrollPoint.y, deltaY );
+        TestMethods.triggerFakeWheelEvent("wheel", svg, scrollPoint.x, scrollPoint.y, deltaY);
```

Comment:
it would be clearer if you can fake same `WheelEvent` before setting `minDomainDxtent` and see it goes beyond that, and reset domain and does it again and see it doesn't go beyond that
I am reading this code and I am not 100% sure if it passes because `minDomainExtent` works or if you didn't zoom enough (this one is obvious that you wheel a lot, but the one below with pitching just looks magical to me)